### DISCUSSION TITLE
How to delete plugin uploaded by myself.

### DISCUSSION THREAD

**Discussion by lukemin** [Difegue/LANraragi#380] [Q&A] [answered]
**LRR Version and OS**
0.7.5 docker
__________
I try to modify hdoujin plunin to collect metainfo for info.txt generated by Hitomi downloader.

After I uploaded the modified one, I found it did not work. So when I tried to remove it, but I did't kown how to do that.

**Comment by Difegue** [comment, marked as answer]
There's no built-in functionality for deleting plugin files. You can either:  

* Reupload a plugin with the same name and it'll mercilessly replace it
* Scrap your Docker container and make a new one, which should bring no loss of data if your database/content folder are correctly mapped to external files.
